hudi-utilities/src/test/java/org/apache/hudi/utilities/functional/TestHoodieDeltaStreamer.java

```diff
@@ -1624,6 +1624,13 @@ public TestGenerator(TypedProperties props) {
     }
   }
 
+  public static class TestTableLevelGenerator extends SimpleKeyGenerator {
```

Comment:
Why do we need this new class? `TestGenerator` already exists in `TestHoodieDeltaStreamer`. See if you can use the same.

Reply:
I saw `TestHoodieDeltaStreamer`, but I hope hava a diffrient name and same logic class，because this not affect the previous test and meet my needs.